src/redux/actions/storeLocation.actions.js

```diff
@@ -1,16 +1,29 @@
 export const STORE_LOCATION_ACTIONS = {
   GET_STORES_BY_PRODUCT_IDS: "GET_STORES_BY_PRODUCT_IDS",
-  SEND_STORES_BY_PRODUCT_IDS: "SEND_STORES_BY_PRODUCT_IDS"
-}
+  SEND_STORES_BY_PRODUCT_IDS: "SEND_STORES_BY_PRODUCT_IDS",
+  GET_USER_CURRENT_LOCATION: "GET_USER_CURRENT_LOCATION",
+  SET_USER_CURRENT_LOCATION: "SET_USER_CURRENT_LOCATION",
+};
 
-export const getStoresByProductIDs = ids => ({
+export const getStoresByProductIDs = payload => ({
```

Comment:
I like the name ids better, it's more descriptive and consistent with what we did below. Maybe we could call it `productIDs` instead of payload?